webapp/src/testUtils.tsx

```diff
@@ -24,7 +24,7 @@ export function mockDOM(): void {
     }
 }
 
-export function mockMatchMedia(result: any): void {
+export function mockMatchMedia(result: unknown): void {
```

Comment:
Hello!

I believe this type can be more specific

```suggestion
export function mockMatchMedia(result: {matches: boolean}): void {
```

If we look at were the function is being called, this is the only thing being passed into it.

Reply:
You are right. I applied the suggestion.